.github/workflows/roxctl.yml

```diff
@@ -0,0 +1,33 @@
+name: Roxctl Image Scan
+
+on:
+  schedule:
+    - cron: '0 2 * * *'
+
+permissions:
+  contents: read
+
+jobs:
+  roxctl:
+    permissions:
+      contents: read # for actions/checkout to fetch code
+      security-events: write # for github/codeql-action/upload-sarif to upload SARIF results
+    runs-on: ubuntu-latest
+    steps:
+    - name: Download roxctl
+      run: curl -sSLO https://mirror.openshift.com/pub/rhacs/assets/latest/bin/Linux/roxctl && chmod +x roxctl
+    - name: Image scan
+      env:
+        ROX_API_TOKEN: ${{ secrets.STACKROX_CI_INSTANCE_API_KEY }}
+        ROX_CENTRAL_ADDRESS: ${{ secrets.STACKROX_CI_INSTANCE_CENTRAL_HOST }}
+      run: ./roxctl image scan -e $ROX_CENTRAL_ADDRESS:443 -i quay.io/stackrox-io/main -o table
+    - name: Image check
+      env:
+        ROX_API_TOKEN: ${{ secrets.STACKROX_CI_INSTANCE_API_KEY }}
+        ROX_CENTRAL_ADDRESS: ${{ secrets.STACKROX_CI_INSTANCE_CENTRAL_HOST }}
+      run: ./roxctl image check -e $ROX_CENTRAL_ADDRESS:443 -i quay.io/stackrox-io/main -o junit > check.xml && ./roxctl image check -e $ROX_CENTRAL_ADDRESS:443 -i quay.io/stackrox-io/main -o table
```

Comment:
Will you do this for `quay.io/stackrox-io/roxctl` and `quay.io/stackrox-io/central-db` also?